tests/system/Helpers/TextHelperTest.php

```diff
@@ -27,8 +27,8 @@ public function testStripSlashes()
             "No, my name is O'connor.",
         ];
         $str = [
-            "Is your name O\'reilly?",
-            "No, my name is O\'connor.",
+            "Is your name O\\'reilly?",
+            "No, my name is O\\'connor.",
```

Comment:
Does this actually change the test case? I think not, but want a second opinion.

Reply:
See https://3v4l.org/lk32I